opm/io/eclipse/OutputStream.cpp

```diff
@@ -588,6 +588,9 @@ namespace {
         const auto substrLength    = std::size_t{8};
         const auto maxSubstrings   = std::size_t{9};
         const auto maxStringLength = maxSubstrings * substrLength;
+        // If the length of the path is 72 < N <= 132, 17 8-character words are used
+        const auto maxSubstrings2  = std::size_t{17};
+        const auto maxStringLength2 = std::size_t{132};
```

Comment:
It's mostly academic and I haven't really thought about it, but maybe we could consider switching to calculating the number of substrings in terms of the maximum string size instead of the other way around?  E.g.,
```C++
const auto maxStringLength = std::array {
    std::string::size_type{72},
    std::string::size_type{132}
};

auto numSubStr = [&maxSubStr, subStrLength](const std::size_t i)
{
    return maxSubStr[i] / subStrLength + (maxSubStr[i] % subStrLength > 0);
};

if (restart.root.empty()) {
    ret.resize(numSubStr(0));
    return ret;
}
    
if (restart.root.size() > maxStringLength.back()) {
    // Handle long strings
}

ret.resize(numSubStr(restart.root.size() > maxStringLength.front()));

// ...
```

Reply:
Certainly possible. Since there are here only two specific alternatives depending on the restart path length I think I might prefer to be more explicit - example incoming (but no strong opinion..)